.npmignore

```diff
@@ -1,2 +1,13 @@
-example/
```

Comment:
Instead of a big `.npmignore` to maintain just use the `files` property in your `package.json`